ouroboros-consensus-byron-test/src/Ouroboros/Consensus/ByronDual/Node.hs

```diff
@@ -18,8 +18,8 @@ import           Data.Maybe (fromMaybe)
 
 import qualified Byron.Spec.Ledger.Core as Spec
 import qualified Byron.Spec.Ledger.Delegation as Spec
-import qualified Byron.Spec.Ledger.Update as Spec
 import qualified Byron.Spec.Ledger.UTxO as Spec
+import qualified Byron.Spec.Ledger.Update as Spec
 
```

Comment:
Looks like the old comparison ignored case. (Not list in stylish-haskell CHANGELOG.)